dev/jarvis_app.py

```diff
@@ -0,0 +1,158 @@
+    # 1) connect Notion
+    # 2) connect Telegram
+    # 3) Do something - add task handler
+
+    # 1) connect Notion
+    notion_client = notion.Client(auth=env("JARVIS_NOTION_TOKEN"))
+
+    # 2) connect Telegram
+    import telegram.ext
+    telegram_updater = telegram.ext.Updater(env("JARVIS_TELEGRAM_TOKEN"))
+
+    # 3) add message handler
+
+    class NotionClient:
+        def __init__(self, token):
+            self.client = notion.Client(auth=token)
+
+        # var1 - message text, tags - tags.
+        def add_task(self, name, content, tags):
+            """
+            Example:
+            /add_task New task to cleanup #home sdf #urgent sdf
+
+            [] Clean up kitchen etc #idea
+            [] clean up desk
+            #2hours
+
+            :param text:
+            :param tags:
+            :return:
+            """
+            pars = {"parent": {"database_id": TASKS_DB_ID},
+                    "properties": {
+                        "Name": {
+                            "title": [
+                                {
+                                    "text": {
+                                        "content": name,
+                                    },
+                                },
+                            ],
+                        },
+                    },
+                    }
+            if content:
+                pars["children"] = [
+                    {
+                        "object": "block",
+                        "type": "paragraph",
+                        "paragraph": {
+                            "rich_text": [
+                                {
+                                    "type": "text",
+                                    "text": {
+                                        "content": content,
+                                    },
+                                },
+                            ],
+                        },
+                    },
+                ]
+            self.client.pages.create(**pars)
+
+
+    def parse_command(full_command):
+        """
+        Parse telegram command format for Jarvis
+
+        Examples:
+        1. Simple command example:
+        >>> parse_command('/idea Test jarvis #link Notion Assistant #blue #important')
+        {'command': '/idea', 'text': 'Test jarvis', 'tags': {'link': 'Notion Assistant', 'blue': None, 'important': None}}
+
+        2. Text only
+        >>> parse_command('Just text with tags #link Notion Assistant #blue #important')
+        {'command': None, 'text': 'Just text with tags', 'tags': {'link': 'Notion Assistant', 'blue': None, 'important': None}}
+        """
+        output = {}
+
+        # parse command
+        if full_command.startswith('/'):
+            command, message = full_command.split(None, 1)
+            output['command'] = command.rstrip()
+        else:
+            message = full_command
+            output['command'] = None
+
+        text, *tags = message.split('#')
```

Comment:
I changed my mind.
Let's only parse the tags from the first line of the message

So that if the message will be like
"Jarvis #project
setup env #todo
only parse first line #idea
"

- we don't apply all the tags to the main block